app/models/krikri/original_record_metadata.rb

```diff
@@ -0,0 +1,14 @@
+module Krikri
+  ##
+  # Descriptive Metadata for OriginalRecord
+  class OriginalRecordMetadata < ActiveTriples::Resource
```

Comment:
Might be worth a chat with Gretchen and Amy if we want to call this out in a future version of the DPLA MAP. (Maybe 4.1, just to make the near-term too confusing.)